spec/datadog/core/telemetry/logging_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+require 'datadog/core/telemetry/logging'
+
+RSpec.describe Datadog::Core::Telemetry::Logging do
+  describe '.report' do
+    context 'with named exception' do
+      it 'sends a log event to via telemetry' do
+        telemetry = double('telemetry', log: nil)
```

Comment:
I recommend using an `instance_double` here; regular doubles are quite dangerous since there's nothing that ensures that the double matches the actual class being tested.

Reply:
I am trying to NOT be religious about `instance_double` is safer with 'double'. 

Since they are both not the `Datadog::Core::Telemetry::Component` instance,  I would prefer a mock object with minimized footprint (I can also replaced it with an `OpenStruct`).

Reply:
I'm curious to know why you don't like `instance_double`; I've sent you a message, as I'd like to chat about this.